docs/content/en/docs/implementing/otel.md

```diff
@@ -4,26 +4,193 @@ description: How to standardize access to OpenTelemetry observability data
 weight: 140
 ---
 
+
+The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable.
+In other words, it creates a distributed, end-to-end trace
+of everything Kubernetes does in the context of a Deployment.
```

Comment:
"everything" is too strong 😅 I would soften a bit the language to don't oversell 
```suggestion
of what Kubernetes does in the context of a Deployment.
```